### ISSUE TITLE
Uncaught SyntaxError: Invalid or unexpected token

### ISSUE THREAD

**Issue by qwertytam** [nteract/hydrogen#1854]
### Steps to Reproduce
1. Install Hydrogen from Atom

**Atom**: 1.42.0 x64
**Electron**: 4.2.7
**OS**: Microsoft Windows 10 Home
**Thrown From**: [Hydrogen](https://github.com/nteract/hydrogen) package 2.13.1

### Stack Trace

Uncaught SyntaxError: Invalid or unexpected token

```
At C:\Users\Tom\AppData\Local\atom\app-1.42.0\resources\app\static\<embedded>:11

C:\Users\Tom\.atom\packages\Hydrogen\node_modules\github-markdown-css\github-markdown.css:1
(function (exports, require, module, __filename, __dirname, process, global, Buffer) { return function (exports, require, module, __filename, __dirname) { @font-face {
                                                                                                                                                           ^

SyntaxError: Invalid or unexpected token
    at new Script (vm.js:79:7)
    at NativeCompileCache.runInThisContext (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:146322)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:147206)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:146745)
    at /packages/Hydrogen/node_modules/@nteract/markdown/lib/markdown-render.js:18:1)
    at /packages/Hydrogen/node_modules/@nteract/markdown/lib/markdown-render.js:31:3)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:147429)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:146745)
    at /packages/Hydrogen/node_modules/@nteract/markdown/lib/index.js:6:43)
    at /packages/Hydrogen/node_modules/@nteract/markdown/lib/index.js:11:3)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:147429)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:146745)
    at /packages/Hydrogen/node_modules/@nteract/outputs/lib/components/media/markdown.js:13:36)
    at /packages/Hydrogen/node_modules/@nteract/outputs/lib/components/media/markdown.js:27:3)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:147429)
```

**Comment by JimmyDeSoto** [comment]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.42.0 x64
**Electron**: 4.2.7
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [Hydrogen](https://github.com/nteract/hydrogen) package 2.13.1


### Stack Trace

Uncaught SyntaxError: Invalid or unexpected token

```
At C:\Users\wilba\AppData\Local\atom\app-1.42.0\resources\app\static\<embedded>:11

C:\Users\wilba\.atom\packages\Hydrogen\node_modules\github-markdown-css\github-markdown.css:1
(function (exports, require, module, __filename, __dirname, process, global, Buffer) { return function (exports, require, module, __filename, __dirname) { @font-face {
                                                                                                                                                           ^

SyntaxError: Invalid or unexpected token
    at new Script (vm.js:79:7)
    at NativeCompileCache.runInThisContext (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:146322)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:147206)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:146745)
    at /packages/Hydrogen/node_modules/@nteract/markdown/lib/markdown-render.js:18:1)
    at /packages/Hydrogen/node_modules/@nteract/markdown/lib/markdown-render.js:31:3)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:147429)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:146745)
    at /packages/Hydrogen/node_modules/@nteract/markdown/lib/index.js:6:43)
    at /packages/Hydrogen/node_modules/@nteract/markdown/lib/index.js:11:3)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:147429)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:146745)
    at /packages/Hydrogen/node_modules/@nteract/outputs/lib/components/media/markdown.js:13:36)
    at /packages/Hydrogen/node_modules/@nteract/outputs/lib/components/media/markdown.js:27:3)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:147429)
```

### Commands

```
     -9:13.4.0 application:open (atom-workspace.workspace.scrollbars-visible-always.theme-solarized-dark-syntax.theme-atom-dark-ui)
```

### Non-Core Packages

```
atom-material-syntax 1.0.8 
Hydrogen 2.13.1 
ide-python 1.5.0 
kite 0.176.0 
language-ini 1.23.0 
python-tools 0.6.9 
zentabs 0.8.9 
```

**Comment by wadethestealth** [comment]
Thank you for reporting. Please temporarily use hydrogen 2.8.0 while I fix this install issue.

`apm install hydrogen@2.8.0`

**Comment by qwertytam** [comment]
Thank you - quick question, I have downgraded my python version from 3.8.0 to 3.7.6 and now receive this error `C:\Users\...\Python\Python38\python.exe is not recognized as an internal or external command, operable program or batch file` --> Is this an error being thrown by Hydrogen specifically not finding the change in Python version? Any suggestions on how to fix? Sorry for the newbie questions, but just starting to use Atom / Hydrogen

**Comment by wadethestealth** [comment]
@quertytam It has to do with your python path. Most of the time this is directly pulled from your system/user environment variable `PYTHONPATH` most likely you could change that 38 to a 37 as that is probably the correct directory, but it really just depends on how you installed python as there are a million different ways. But poython does have docs about the pythonpath var.

**Comment by SemanticBeeng** [comment]
[Enter steps to reproduce:]

1. Fresh Atom install
2. Setup packages below with `apm`
3. `Ctrl+Alt+Enter` in Python file


**Atom**: 1.42.0 x64
**Electron**: 4.2.7
**OS**: Ubuntu 16.04.2
**Thrown From**: [Hydrogen](https://github.com/nteract/hydrogen) package 2.13.1


### Stack Trace

Failed to activate the Hydrogen package

```
At Invalid or unexpected token

/home/username/.atom/packages/Hydrogen/node_modules/github-markdown-css/github-markdown.css:1
(function (exports, require, module, __filename, __dirname, process, global, Buffer) { return function (exports, require, module, __filename, __dirname) { @font-face {
                                                                                                                                                           ^

SyntaxError: Invalid or unexpected token
    at new Script (vm.js:79:7)
    at NativeCompileCache.runInThisContext (/usr/share/atom/resources/app/static/<embedded>:11:146322)
    at Module.get_Module._compile (/usr/share/atom/resources/app/static/<embedded>:11:147206)
    at Object.value [as .js] (/usr/share/atom/resources/app/static/<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (/usr/share/atom/resources/app/static/<embedded>:11:146745)
    at /packages/Hydrogen/node_modules/@nteract/markdown/lib/markdown-render.js:18:1)
    at /packages/Hydrogen/node_modules/@nteract/markdown/lib/markdown-render.js:31:3)
    at Module.get_Module._compile (/usr/share/atom/resources/app/static/<embedded>:11:147429)
    at Object.value [as .js] (/usr/share/atom/resources/app/static/<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (/usr/share/atom/resources/app/static/<embedded>:11:146745)
    at /packages/Hydrogen/node_modules/@nteract/markdown/lib/index.js:6:43)
    at /packages/Hydrogen/node_modules/@nteract/markdown/lib/index.js:11:3)
    at Module.get_Module._compile (/usr/share/atom/resources/app/static/<embedded>:11:147429)
    at Object.value [as .js] (/usr/share/atom/resources/app/static/<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (/usr/share/atom/resources/app/static/<embedded>:11:146745)
    at /packages/Hydrogen/node_modules/@nteract/outputs/lib/components/media/markdown.js:13:36)
    at /packages/Hydrogen/node_modules/@nteract/outputs/lib/components/media/markdown.js:27:3)
    at Module.get_Module._compile (/usr/share/atom/resources/app/static/<embedded>:11:147429)
    at Object.value [as .js] (/usr/share/atom/resources/app/static/<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (/usr/share/atom/resources/app/static/<embedded>:11:146745)
...
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (internal/modules/cjs/helpers.js:20:18)
    at customRequire (/usr/share/atom/resources/app/static/<embedded>:1:689402)
    at Package.requireMainModule (/usr/share/atom/resources/app/static/<embedded>:11:3220707)
    at Package.activateNow (/usr/share/atom/resources/app/static/<embedded>:11:3208691)
    at activationCommandSubscriptions.add.commandRegistry.onWillDispatch.o (/usr/share/atom/resources/app/static/<embedded>:14:1879)
    at Function.simpleDispatch (/usr/share/atom/resources/app/static/<embedded>:11:1211760)
    at Emitter.emit (/usr/share/atom/resources/app/static/<embedded>:11:1213201)
    at CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/static/<embedded>:11:349017)
    at KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app/static/<embedded>:11:1252387)
    at KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app/static/<embedded>:11:1248521)
    at WindowEventHandler.handleDocumentKeyEvent (/usr/share/atom/resources/app/static/<embedded>:11:284591)
```

### Commands

```
     -2:10.3.0 hydrogen:run-cell (input.hidden-input)
```

### Non-Core Packages

```
atom-ide-ui 0.6.0 
Hydrogen 2.13.1 
hydrogen-launcher 1.2.2 
hydrogen-python 0.0.8 
ide-python 1.5.0 
MagicPython 1.1.1 
script 3.25.0 
```

The `font-face` section looks like this

```
@font-face {
  font-family: octicons-link;
  src: url(data:font/woff;charset=utf-8;base64,d09GRgABAAAAAAZwABAAAAAACFQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABEU0lHAAAGaAAAAAgAAAAIAAAAAUdTVUIAAAZcAAAACgAAAAoAAQAAT1MvMgAAAyQAAABJAAAAYFYEU3RjbWFwAAADcAAAAEUAAACAAJThvmN2dCAAAATkAAAABAAAAAQAAAAAZnBnbQAAA7gAAACyAAABCUM+8IhnYXNwAAAGTAAAABAAAAAQABoAI2dseWYAAAFsAAABPAAAAZwcEq9taGVhZAAAAsgAAAA0AAAANgh4a91oaGVhAAADCAAAABoAAAAkCA8DRGhtdHgAAAL8AAAADAAAAAwGAACfbG9jYQAAAsAAAAAIAAAACABiATBtYXhwAAACqAAAABgAAAAgAA8ASm5hbWUAAAToAAABQgAAAlXu73sOcG9zdAAABiwAAAAeAAAAME3QpOBwcmVwAAAEbAAAAHYAAAB/aFGpk3jaTY6xa8JAGMW/O62BDi0tJLYQincXEypYIiGJjSgHniQ6umTsUEyLm5BV6NDBP8Tpts6F0v+k/0an2i+itHDw3v2+9+DBKTzsJNnWJNTgHEy4BgG3EMI9DCEDOGEXzDADU5hBKMIgNPZqoD3SilVaXZCER3/I7AtxEJLtzzuZfI+VVkprxTlXShWKb3TBecG11rwoNlmmn1P2WYcJczl32etSpKnziC7lQyWe1smVPy/Lt7Kc+0vWY/gAgIIEqAN9we0pwKXreiMasxvabDQMM4riO+qxM2ogwDGOZTXxwxDiycQIcoYFBLj5K3EIaSctAq2kTYiw+ymhce7vwM9jSqO8JyVd5RH9gyTt2+J/yUmYlIR0s04n6+7Vm1ozezUeLEaUjhaDSuXHwVRgvLJn1tQ7xiuVv/ocTRF42mNgZGBgYGbwZOBiAAFGJBIMAAizAFoAAABiAGIAznjaY2BkYGAA4in8zwXi+W2+MjCzMIDApSwvXzC97Z4Ig8N/BxYGZgcgl52BCSQKAA3jCV8CAABfAAAAAAQAAEB42mNgZGBg4f3vACQZQABIMjKgAmYAKEgBXgAAeNpjYGY6wTiBgZWBg2kmUxoDA4MPhGZMYzBi1AHygVLYQUCaawqDA4PChxhmh/8ODDEsvAwHgMKMIDnGL0x7gJQCAwMAJd4MFwAAAHjaY2BgYGaA4DAGRgYQkAHyGMF8NgYrIM3JIAGVYYDT+AEjAwuDFpBmA9KMDEwMCh9i/v8H8sH0/4dQc1iAmAkALaUKLgAAAHjaTY9LDsIgEIbtgqHUPpDi3gPoBVyRTmTddOmqTXThEXqrob2gQ1FjwpDvfwCBdmdXC5AVKFu3e5MfNFJ29KTQT48Ob9/lqYwOGZxeUelN2U2R6+cArgtCJpauW7UQBqnFkUsjAY/kOU1cP+DAgvxwn1chZDwUbd6CFimGXwzwF6tPbFIcjEl+vvmM/byA48e6tWrKArm4ZJlCbdsrxksL1AwWn/yBSJKpYbq8AXaaTb8AAHja28jAwOC00ZrBeQNDQOWO//sdBBgYGRiYWYAEELEwMTE4uzo5Zzo5b2BxdnFOcALxNjA6b2ByTswC8jYwg0VlNuoCTWAMqNzMzsoK1rEhNqByEyerg5PMJlYuVueETKcd/89uBpnpvIEVomeHLoMsAAe1Id4AAAAAAAB42oWQT07CQBTGv0JBhagk7HQzKxca2sJCE1hDt4QF+9JOS0nbaaYDCQfwCJ7Au3AHj+LO13FMmm6cl7785vven0kBjHCBhfpYuNa5Ph1c0e2Xu3jEvWG7UdPDLZ4N92nOm+EBXuAbHmIMSRMs+4aUEd4Nd3CHD8NdvOLTsA2GL8M9PODbcL+hD7C1xoaHeLJSEao0FEW14ckxC+TU8TxvsY6X0eLPmRhry2WVioLpkrbp84LLQPGI7c6sOiUzpWIWS5GzlSgUzzLBSikOPFTOXqly7rqx0Z1Q5BAIoZBSFihQYQOOBEdkCOgXTOHA07HAGjGWiIjaPZNW13/+lm6S9FT7rLHFJ6fQbkATOG1j2OFMucKJJsxIVfQORl+9Jyda6Sl1dUYhSCm1dyClfoeDve4qMYdLEbfqHf3O/AdDumsjAAB42mNgYoAAZQYjBmyAGYQZmdhL8zLdDEydARfoAqIAAAABAAMABwAKABMAB///AA8AAQAAAAAAAAAAAAAAAAABAAAAAA==) format('woff');
}
```

UPDATE: Removed the `font-face` section and now getting this
```
At Unexpected token .

/home/nickdsc/.atom/packages/Hydrogen/node_modules/github-markdown-css/github-markdown.css:1
(function (exports, require, module, __filename, __dirname, process, global, Buffer) { return function (exports, require, module, __filename, __dirname) { .markdown-body .octicon {
                                                                                                                                                           ^

SyntaxError: Unexpected token .
    at new Script (vm.js:79:7)
    at NativeCompileCache.runInThisContext (/usr/share/atom/resources/app/static/<embedded>:11:146322)
    at Module.get_Module._compile (/usr/share/atom/resources/app/static/<embedded>:11:147206)
    at Object.value [as .js] (/usr/share/atom/resources/app/static/<embedded>:11:150977)
```

### Commands

```
  3x -0:20 core:close (atom-pane.pane.active)
     -0:15.9.0 hydrogen:run (input.hidden-input)
```

### Non-Core Packages

```
Hydrogen 2.13.1 
```

**Comment by mechloz** [comment]
Same issue here

**Comment by jk31** [comment]
Same here, fresh python 3.7.6 with a new installation of Atom

**Comment by qwertytam** [comment]
@wadethestealth re python path, I've installed and successfully run a simple 'hello world' script using the Atom package `run-python-simply v0.6.0`. My hypotheses is that `Hydrogen` is for some reason thinking that `python v3.8.0` is still installed. I have been through my path settings, program folders, registry settings etc. to try and find any stray links to `python v3.8.0`. Any suggestions on how to test if `Hydrogen` is picking up `python v3.8.0` instead of `python v3.7.6`?

**Comment by bobcolner** [comment]
new user here -I have the same issue with a fresh install

**Comment by wadethestealth** [comment]
@jk31 that is a separate issue from this one. Normally this is caused by a program using or holding that file as a resource, most commonly disabling antivirus on hydrogen will solve most instances. @qwertytam then it is because your kernel.json has your kernel pointing to that python38. navigate to your kernel.json file from jupyter and modify the path to be the correct path.

**Comment by qwertytam** [comment]
@wadethestealth thank you! That fixed my issue. `kernel.json` is located in `C:\Users\...\AppData\Roaming\jupyter\kernels\python3` for those looking to locate it.

**Comment by SilvesterHsu** [comment]
Same issue with Atom(1.42.0), python(3.7.4) and Hydrogen(2.13.1)

**Comment by paulathomas** [comment]
I've just hit the same problem on ubuntu 19.10 with python 3.7 and hydrogen 2.13, am attempting downgrade to 2.8 of hydrogen.

**Comment by xiaofengzhiyu** [comment]
same issue here

**Comment by aviatesk** [comment]
Close since this will be fixed in Hydrogen@2.14.0
